importer/Monument.py

```diff
@@ -295,7 +295,7 @@ def print_datafiles_titles(self):
         for title in self.data_files:
             print(title)
 
-    def __init__(self, db_row_dict, mapping, data_files, existing):
+    def __init__(self, db_row_dict, mapping, data_files, existing, repository):
```

Comment:
Does this have a docstring and if so does that mention the repository parameter? (Hard to say with all of the parallel patches)